### ISSUE TITLE
 No grant permission

### ISSUE THREAD

**Issue by rkbansal** [coreorm/s3-nosql#1]
It is saying Access Denied even if my Bucket is open for everyone.

**Comment by coreorm** [comment]
yeah it's so weird! i reckon if you register with packagist and put it there as a public package it might solve this issue.
